src/local-server/controller.ts

```diff
@@ -60,6 +60,11 @@ export class Controller {
   }
 
   private async updateScreenshots(test: Test) {
+    // allow for empty baseline
+    if (!test.baselinePath) {
```

Comment:
Since we rely on the current path pattern here, could you add a TODO comment as a heads up?

Reply:
Added a comment